episodes/4-advanced-layer-types.Rmd

```diff
@@ -27,45 +27,35 @@ So far, we have seen one type of layer, namely the **fully connected**, or **den
 However, there are many different types of layers that perform different calculations and take different inputs. In this episode we will take a look at **convolutional layers** and **dropout layers**, which are useful in the context of image data, but also in many other types of (structured) data.
 
 ## 1. Formulate / Outline the problem: Image classification
-Keras comes with a few prepared datasets. We have a look at the [CIFAR10 dataset](https://www.cs.toronto.edu/~kriz/cifar.html),
-which is a widely known dataset for image classification.
-```python
-from tensorflow import keras
-(train_images, train_labels), (val_images, val_labels) = keras.datasets.cifar10.load_data()
-```
-
-::: callout
-## CERTIFICATE_VERIFY_FAILED error when downloading CIFAR-10 dataset
-When loading the CIFAR-10 dataset, you might get the following error:
-```
-[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1125)
-```
-You can solve this error by adding this to your notebook:
-```python
-import ssl
-ssl._create_default_https_context = ssl._create_unverified_context
-```
-:::
-
-## CIFAR-10
-
-The CIFAR-10 dataset consists of images of 10 different classes: airplanes, cars, birds, cats, deer, dogs, frogs, horses, ships, and trucks.
-It is widely used as a benchmark dataset for image classification. The low resolution of the images in the dataset allows for quick loading and testing models.
-
-For more information about this dataset and how it was collected you can check out
-[Learning Multiple Layers of Features from Tiny Images by  Alex Krizhevsky, 2009](https://www.cs.toronto.edu/~kriz/learning-features-2009-TR.pdf).
+The MLCommons Dollar Street Dataset is a collection of images of everyday household items from homes around the world that visually captures socioeconomic diversity of traditionally underrepresented populations. We use a subset of the original dataset that can be used for multiclass classification with 10 categories. Let's load the data:
 
-
-![Sample images from the CIFAR-10 data-set. Each image is labelled with a category, for example: 'frog' or 'horse'](fig/04_cifar10.png){alt="A 5 by 5 grid of 25 sample images from the CIFAR-10 data-set. Each image is labelled with a category, for example: 'frog' or 'horse'."}
-
-We take a small sample of the data as training set for demonstration purposes.
 ```python
-n = 5000
-train_images = train_images[:n]
-train_labels = train_labels[:n]
-```
-
-
+import pathlib
+DATA_FOLDER = pathlib.Path('data/dataset_dollarstreet/') # change to location where you stored the data
+train_images = np.load(DATA_FOLDER / 'train_images.npy')
+val_images = np.load(DATA_FOLDER / 'test_images.npy')
+train_labels = np.load(DATA_FOLDER / 'train_labels.npy')
+val_labels = np.load(DATA_FOLDER / 'test_labels.npy')
+```
+
```

Comment:
```suggestion

:::::::::::::::::::::::::::::::::::::::::: callout

### A Note About Data Provenance
In an earlier version, this part of the lesson used a different example dataset.
During [peer review](https://github.com/carpentries-lab/reviews/issues/25#issuecomment-1953271802), the decision was made to replace that dataset due to the way it had been compiled using images "scraped" from the internet without permission from or credit to the original creators of those images. Unfortunately, uncredited use of images is a common problem among datasets used to benchmark models for image classification.
The Dollar Street Dataset was chosen for use in the lesson as it contains only images [created by the Gapminder project](https://www.gapminder.org/dollar-street/about?) expressly for the purposes of use in the dataset.
The original Dollar Street Dataset is very large -- more than 100 GB -- with the potential to grow even bigger, so we created a subset for use in this lesson.

::::::::::::::::::::::::::::::::::::::::::::::::::


```

Suggesting this to address https://github.com/carpentries-incubator/deep-learning-intro/issues/445#issuecomment-1988375148

Reply:
Love it!!!